spec/features/mediated_deposit_versioning_spec.rb

```diff
@@ -37,6 +37,13 @@
       expect(page).to have_content 'You have successfully submitted your deposit'
       click_link 'Return to dashboard'
 
+      # check collection link in approvals table
+      within_table('Approvals') do
```

Comment:
I'd prefer that we keep this test focused on doing a mediated deposit.  Can this be in a component test instead?

Reply:
FYI, as I mentioned, the analogous Works::LinkToShowComponent only tests that titles are truncated, nothing about the link.